From 1d61b91330e4e78586a7a9c85cdc01e26af2d9db Mon Sep 17 00:00:00 2001
From: Chen Yu Y <yu.y.chen@intel.com>
Date: Tue, 21 Nov 2017 19:38:37 +0800
Subject: [PATCH 10/17] Add missing basic header files for VNDK enabling

When enable "BOARD_VNDK_VERSION:=current",then modules will
reference to "include_vndk" instead of "include", and below 5 header files
are not in "include_vndk" scope. If modules are missing these header
files, it'll build break, so adding header files explicitly is required
to avoid build break

<pthread.h>
<stdint.h>
<stdio.h>
<sys/types.h>
<time.h>
<unistd.h>

Jira: None
Test: Device boots to home screen.
Change-Id: Ifb4f2d8bc1d54c44618f5c0d4bde802355ff044d
Signed-off-by: Chen Yu Y <yu.y.chen@intel.com>
---
 trusty/storage/proxy/ipc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/trusty/storage/proxy/ipc.h b/trusty/storage/proxy/ipc.h
index 2e366bb..994aa6a 100644
--- a/trusty/storage/proxy/ipc.h
+++ b/trusty/storage/proxy/ipc.h
@@ -17,6 +17,7 @@
 
 #include <stdint.h>
 #include <trusty/interface/storage.h>
+#include <unistd.h>
 
 int ipc_connect(const char *device, const char *service_name);
 void ipc_disconnect(void);
-- 
2.7.4

